ngsi_adapter/src/test/acceptance/features/component/send_data/send_data_parsers.feature

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+Feature: Checking implemented parsers
+  As a monitoring element (probe) user
+  I want to be able to transform monitoring data from probes to NGSI context attributes using all defined parsers
+  so that I can constantly check the status and performance of the cloud infrastructure using NGSI Context Broker.
+
+
+  @happy_path
+  Scenario Outline: Probe sends valid raw data using an existing and well-formed parser
+    Given the parser "<parser>"
+    And   the monitored resource with id "<probe_id>" and type "<probe_type>"
+    When  I send raw data according to the selected parser
+    Then  the response status code is "200"
+
+    Examples:
+    | parser       | probe_id       | probe_type |
+    | check_disk   | qa:192.168.1.2 | host       |
+    | check_load   | qa:192.168.1.3 | host       |
+    | check_mem.sh | qa:192.168.1.4 | host       |
+    | check_procs  | qa:192.168.1.5 | host       |
+    | check_users  | qa:192.168.1.6 | host       |
+
+
+  Scenario: Probe sends raw data using a malformed parser
```

Comment:
For instance, I was thinking about having a *parser.js* with code/syntax errors and check the NGSI-Adapter behaviour in this case; or a parser that builds a malformed attribute structure or throws a not controlled exception; or something like this...